Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RenderPipeline.h

```diff
@@ -201,6 +214,19 @@ namespace AZ
             //! use RPI::PassSystemInterface::Get()->ForEachPass() function instead.
             Ptr<Pass> FindFirstPass(const AZ::Name& passName);
 
+            //! Finding, adding and removing pipeline attachments (attachments that can be referenced directly by any pass in the pipeline)
+            const Ptr<PassAttachment> GetPipelineAttachment(const Name& name) const;
+            void AddPipelineAttachment(const Ptr<PassAttachment>& attachment);
+            void RemovePipelineAttachment(const Ptr<PassAttachment>& attachment);
+
+            //! Finding, adding and removing pipeline connections/bindings (can be referenced directly by any pass in the pipeline)
+            const GlobalBinding* GetPipelineConnection(const Name& name) const;
```

Comment:
This is a perfect example: 'GlobalBinding' is returned but the method name is 'GetPipelineConnection' (Pipeline --> Global)

Reply:
So to follow up on Vicky's comment - this can become 'GetGlobalPipelineConnection'